website/content/docs/networking/consul/service-mesh.mdx

```diff
@@ -58,11 +62,12 @@ guide.
 ## Nomad Consul service mesh example
 
 The following section walks through an example to enable secure communication
-between a web dashboard and a backend counting service. The web dashboard and
-the counting service are managed by Nomad. Nomad additionally configures Envoy
-proxies to run along side these applications. The dashboard is configured to
-connect to the counting service via localhost on port 9001. The proxy is managed
-by Nomad, and handles mTLS communication to the counting service.
+between a web dashboard and a backend counting service. Nomad and Consul are
+running in the same datacenter. The web dashboard and the counting service are
```

Comment:
I think by "datacenter" you're referring to "the physical place where the computers go" but given that Nomad has a `datacenter` field and Consul has a `datacenter` field and they unfortunately mean something different, maybe we don't want to use that term here? Or if we do because we want to make sure they're all in the same Consul cluster, we should say "Consul `datacenter`"?

Reply:
Ah, ok. Given that context, the Nomad `datacenter` doesn't come into play at all (it's entirely is invisible to Consul service mesh) and we're definitely talking about the Consul `datacenter`.

Reply:
I did some research this morning, and I'm going to remove the new sentence "Nomad and Consul are running in the same datacenter."  We can have Nomad apps running in separate Nomad datacenters. Each Nomad datacenter is typically associated with a Consul datacenter for service discovery. Consul datacenters are federated, which allows them to share service discovery information.

Reply:
> Each Nomad datacenter is typically associated with a Consul datacenter for service discovery. 

A Nomad _region_ is associated with a Consul datacenter for service discovery. It would be extremely challenging to have a Nomad region (which consists of multiple DCs) split among multiple Consul datacenters. But yeah I agree the sentence doesn't add anything here.